core/che-core-metrics-core/src/main/java/org/eclipse/che/core/metrics/TomcatMetricsProvider.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.core.metrics;
+
+import io.micrometer.core.instrument.Tags;
+import io.micrometer.core.instrument.binder.tomcat.TomcatMetrics;
+import javax.inject.Inject;
+import javax.inject.Provider;
+import javax.inject.Singleton;
+import org.apache.catalina.Manager;
+
+/**
+ * @Provider of @{@link io.micrometer.core.instrument.binder.tomcat.TomcatMetrics} instance. Used
```

Comment:
Why do you wrote `@Provider` here? Should not it be `Provider` or `{@link Provider}`?